pages/project-details/[id]/index.js

```diff
@@ -125,6 +149,12 @@ export default function ProjectDetails({
         }
       />
       <Main>
+        <Button
+          onClick={toggleActiveStatus}
+          className={!currentProject.isActive ? "inactive" : ""}
```

Comment:
you shouldn't use a className here since you can solve it with styled components:

[See this as reference](https://github.com/neuefische/web-hh-22-7/blob/main/sessions/react-styled-components/react-styled-components.md#adapting-based-on-props)